shardingsphere-infra/shardingsphere-infra-context/src/main/java/org/apache/shardingsphere/infra/context/metadata/MetaDataContextsBuilder.java

```diff
@@ -94,7 +93,7 @@ public StandardMetaDataContexts build() throws SQLException {
         for (String each : schemaRuleConfigs.keySet()) {
             mataDataMap.put(each, buildMetaData(each));
         }
-        return new StandardMetaDataContexts(mataDataMap, buildGlobalSchemaMetaData(mataDataMap), executorEngine, new ShardingSphereUsers(users), props);
+        return new StandardMetaDataContexts(mataDataMap, buildGlobalSchemaMetaData(mataDataMap), executorEngine, new ShardingSphereUsers(getUsersFromAuthorityRule()), props);
```

Comment:
`ShardingSphereUsers(users)` is expected to be removed from `StandardMetaDataContexts`. That way, `getUsersFromAuthorityRule ()` and `shardingsphere-infra-authority-api` of `shardingsphere-infra/shardingsphere-infra-context/pom.xml` can be deleted together. 

Finally, all the content concerning `authority` and `users` will be restricted in the `Authority` global rule. That's our ideal result.


@zhujunxxxxx This PR looks great really. I did not imagine we can merge it one! Thanks for your prompt nice work.
@huanghao495430759 Please give a glance at this comment. Your issue has no block, go ahead pacing yourself.

Reply:
@tristaZero 
> ShardingSphereUsers(users) is expected to be removed from StandardMetaDataContexts
it's need remove? Ok, next PR I will remove it.